sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1683,9 +1683,11 @@ def provision_with_retries(
                 if not to_provision.cloud.supports(self._requested_features):
                     self._blocked_resources.add(
                         resources_lib.Resources(cloud=to_provision.cloud))
+                    requested_features_str = ', '.join(
+                        [f.value for f in self._requested_features])
                     raise exceptions.ResourcesUnavailableError(
                         f'{to_provision.cloud} does not support: '
-                        f'{self._requested_features}.')
+                        f'{requested_features_str}.')
```

Comment:
nit: How about `... does not support all the features in {requested_features_str}`?

Reply:
Updated!